File: LoanManagerAPI_V2/LoanManager.Api/Models/UserRegistration.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LoanManager.Api.Models
{
    public class UserRegistration
    {
        public string FirstName { get; set; }

        public string MiddleName { get; set; }

        public string LastName { get; set; }

        public string Email { get; set; }

        public string Password { get; set; }

        public string ConfirmPassword { get; set; }

        public bool Agree { get; set; }
    }
}

File: CustomerPortalUI_V1/src/app/core/models/user-management/user-registration.ts
export class UserRegisration {
    FirstName: string;
    MiddleName: string;
    LastName: string;
	Email: string;
    Password: string;
    ConfirmPassword: string;
    Agree: boolean;

    constructor() {
        this.FirstName = "";
        this.MiddleName = "";
        this.LastName = "";
        this.Email = "";
        this.Password = "";
        this.ConfirmPassword = "";
        this.Agree = false;
    }
}